### PR TITLE
0548 - Simple CTA FE

### PR DESCRIPTION
**Pull request by chris-lawton**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1363630548)

### Description of Changes Made

- Adds the FE for the simple CTA block - the template matches the BE work done in https://github.com/torchbox/torchbox.com/pull/30
- Adds a `--color--streamfield-background` var which will also be used on images on the upcoming blog page.

### How to Test

Viewable in the pattern library on the components page: http://localhost:8000/pattern-library/pattern/patterns/styleguide/components/components.html

Or in isolation: http://localhost:8000/pattern-library/pattern/patterns/molecules/streamfield/blocks/call_to_action.html

Or on the standard page: http://localhost:8000/pattern-library/render-pattern/patterns/pages/standard/standard_page.html

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-01-17 at 10 20 55

Screenshot 2024-01-17 at 10 21 03

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Tests and linting passes.

#### Unit tests

- [ ] Added
- [ ] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [ ] Not required

#### Browser testing

- [ ] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [ ] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [ ] Automated WCAG 2.1 tests pass
- [ ] Manual WCAG 2.1 tests completed
- [ ] I have tested in a screen reader
- [ ] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [ ] New font variants have not been added
- [ ] Not required

#### Pattern library

- [ ] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/static_src/sass/components/_call-to-action.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @include media-query(large) {
+        padding: $spacer-medium;
+    }
+
+    &__inner {
+        @include media-query(large) {
+            display: flex;
+            align-items: flex-end;
+            justify-content: space-between;
+            gap: 40px;
```
Comment (helenb) [inline tbx/static_src/sass/components/_call-to-action.scss:16]: nitpick: you could use a spacer var here (2 * $spacer-mini-plus) - but maybe that is just a bit overkill as it starts to mean less in terms of the spacing scheme.

File: tbx/static_src/sass/components/_call-to-action.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    }
+
+    &__text {
+        @include font-size(heading-four);
+        color: var(--color--heading);
+        max-width: 720px;
```
Comment (helenb) [inline tbx/static_src/sass/components/_call-to-action.scss:23]: Query: is this max-width used for other streamfields anywhere / related to any other widths? It looks like it needs to align on the right with the paragraph content above - just wondering if a variable can be used?

**Review by helenb — APPROVED** [review]
Looks great, thanks @chris-lawton . Happy to merge straight away - comments were only v small.

File: tbx/static_src/sass/components/_call-to-action.scss (hunk 2 above)
Comment (chris-lawton) [reply tbx/static_src/sass/components/_call-to-action.scss:23]: The other streamfields widths are controlled by the grid but this component is unique in that it extends to the edge of the screen. There may be more instances so i'll assign it to a variable - more chance other devs will follow suit then : )

Comment (chris-lawton) [reply tbx/static_src/sass/components/_call-to-action.scss]: https://github.com/torchbox/torchbox.com/pull/36#:~:text=to%20a%20var-,e0f7dd0,-Merge%20state

**Pull request merged**